### PR TITLE
Update instructions for setting up Docker Runner

### PR DESCRIPTION
**Pull request by hemanthmantri**
- Added instructions for setting via visual editor
- Added how to bypass mac security setting

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 1]
```diff
@@ -20,41 +20,39 @@ You can define a CI build infrastructure on any Linux or MacOS host. This is the
...
-	2. *MacOS only —* Add this setting to the `environment` list:  
+	2. *macOS only —* Add this setting to the `environment` list:  
 	`- RUNNER_URL=`[`http://host.docker.internal:3000`](http://host.docker.internal:3000/)
 	3. *Linux only —* Add the following line immediately after `restart: unless-stopped"`:  
 	`network_mode: "host"`
 7. To start the delegate, cd to the folder where you downloaded the docker-compose file and then enter:  
  `docker-compose -f <`*`updated-docker-compose.yml`*`> up -d`
 8. Download a Drone Runner executable from <https://github.com/harness/drone-docker-runner/releases> .
-9. Enable execution permissions for the Runner. For example, on MacOS you can run the command `sudo` `chmod +x` `drone-docker-runner-darwin-amd64`.
+9. Enable execution permissions for the Runner. For example, on macOS you can run the command `sudo` `chmod +x` `drone-docker-runner-darwin-amd64`.
```
Comment (happyhebaby) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:38]: This is for linux. In macOS, besides doing this, you also need to allow the application to be opened by following this link: https://stackoverflow.com/questions/60112375/apple-cannot-check-it-for-malicious-software-in-every-installation

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 2]
```diff
@@ -20,41 +20,39 @@ You can define a CI build infrastructure on any Linux or MacOS host. This is the
 
 ### Install the Delegate
 
-1. in the Harness UI, go to the screen that corresponds to the scope of your delegate: **Account Settings,** **Account Settings > Organizations**, or **Project Setup**.
-2. Click **New Delegate**, then select **Docker** for the delegate type and click **Continue**.
+1. In the Harness UI, go to the screen that corresponds to the scope of your delegate: **Account Settings > Account Resources > Delegates **, or **Project Setup > Delegates**.
+2. Click **New Delegate** or **Create a Delegate**, and select **Docker** for the delegate type and click **Continue**.
```
Comment (KimberlyFields) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:24]:
```suggestion
2. Select **New Delegate** or **Create a Delegate**, select **Docker** for the delegate type, and then select **Continue**.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 3]
```diff
@@ -20,41 +20,39 @@ You can define a CI build infrastructure on any Linux or MacOS host. This is the
...
 	3. *Linux only —* Add the following line immediately after `restart: unless-stopped"`:  
 	`network_mode: "host"`
 7. To start the delegate, cd to the folder where you downloaded the docker-compose file and then enter:  
  `docker-compose -f <`*`updated-docker-compose.yml`*`> up -d`
 8. Download a Drone Runner executable from <https://github.com/harness/drone-docker-runner/releases> .
-9. Enable execution permissions for the Runner. For example, on MacOS you can run the command `sudo` `chmod +x` `drone-docker-runner-darwin-amd64`.
+9. Enable execution permissions for the Runner. For example, on macOS you can run the command `sudo` `chmod +x` `drone-docker-runner-darwin-amd64`.
 10. Start the runner binary:
-	1. On MacOS, run `./drone-docker-runner-darwin-amd64 server`
+	1. On macOS, run `./drone-docker-runner-darwin-amd64 server` (Note: You might have to allow this app to be run from `Security and Privacy` settings)
```
Comment (KimberlyFields) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:40]:
```suggestion
	1. On macOS, run `./drone-docker-runner-darwin-amd64 server` Note: You might have to allow this app to be run from the**Security and Privacy** settings.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 4]
```diff
@@ -20,41 +20,39 @@ You can define a CI build infrastructure on any Linux or MacOS host. This is the
...
 	2. On Linux, run as sudo: `sudo ./drone-docker-runner-darwin-amd64 server`
 11. Go back to the Delegates page where you created the new delegate. The delegate should appear as connected. This might take a few minutes.
 
 ### Update the Pipeline
 
-Update the pipeline where you want to use the Docker delegate:
+Update the pipeline where you want to use the Docker delegate either from YAML or the Visual pipeline editor.
```
Comment (KimberlyFields) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:46]:
```suggestion
Update the pipeline where you want to use the Docker delegate, either from YAML or the Visual pipeline editor.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 5]
```diff
@@ -82,9 +79,17 @@ Update the pipeline where you want to use the Docker delegate:
            ...  
 
 ```
+
+#### From the Visual Editor
+
+Update the infrastructure tab to use:
```
Comment (KimberlyFields) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:85]:
```suggestion
Update the **Infrastructure** tab to use:
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md (hunk 1 above)
Comment (KimberlyFields) [reply docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:38]: @hemanthmantri can you address the comment above from happyhebaby so we can get this content published? Thanks.

**Review by KimberlyFields — APPROVED** [review]

**Comment by ravilach** [comment]
@KimberlyFields @hemanthmantri looks like we never merged this one in. All good to go?

**Comment by KimberlyFields** [comment]
Waiting for @hemanthmantri to accept or reject the comment from happyhebaby.

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md (hunk 1 above)
Comment (hemanthmantri) [reply docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:38]: This is mentioned already in bullet #10.

**Comment by hemanthmantri** [comment]
@KimberlyFields: Please approve.  I can take up any follow-up comments later. Thanks.

**Pull request merged**